apps/rd/rd-professional-api/rd-professional-api.yaml

```diff
@@ -7,6 +7,9 @@ spec:
   values:
     java:
       replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 4
       memoryRequests: "1792Mi"
```

Comment:
Could you please update memoryRequests to 2048Mi to align with what we tested in perftest?